system/ee/ExpressionEngine/Addons/pro_search/libraries/Pro_search_filters.php

```diff
@@ -102,16 +102,21 @@ public function __construct()
                 }
 
                 // Compose file name
-                $file = $dir . "/lsf.{$item}.php";
+                // Compose class name
+                if (version_compare(ee()->config->item('app_version'), '7.0.0', '>=')) {
```

Comment:
This line would always return `true` because we're including this only for EE7 :)